test/testvalueflow.cpp

```diff
@@ -105,6 +106,25 @@ class TestValueFlow : public TestFixture {
         TEST_CASE(valueFlowUninit);
     }
 
+    bool testValueOfXKnown(const char code[], unsigned int linenr) {
```

Comment:
please add a 3rd parameter for the value. I would prefer that the `testValueOfX` is not needed also.